storybook/.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
Whats this needed for?

Reply:
This file appeared when I ran webpack & npm migration command from storybook.
But it seems to work without this setting. So I deleted it.